.github/workflows/test_all.yml

```diff
@@ -92,9 +92,9 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           flags: ${{ matrix.operating-system }},${{ matrix.python-version }},${{ matrix.sktime-component }},complete
-  test_components_without_extras:
+  test_module:
```

Comment:
This is the step that uses `all_extras_pandas2`, and doesn't use dependencies only for a specific component/module.